src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,99 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void clearDataBase() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_validUser_ok() {
+        User newUser = new User("Forzen123212", "forzen123212", 20);
+        User registratedUser = registrationService.register(newUser);
+        assertEquals(1, Storage.people.size());
+        assertEquals(newUser, registratedUser);
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        User newUser = null;
+        assertThrows(RegistrationException.class,
+                () -> registrationService.register(newUser));
+    }
+
+    @Test
+    void register_existingUser_notOk() {
+        User newUser = new User("Forzen123212", "forzen123212", 20);
```

Comment:
Try to avoid using "Forzen123212", "forzen123212" strings. Make them more descriptive or make them as constants.